frontend/webapp/components/side.menu/menu/items.tsx

```diff
@@ -0,0 +1,32 @@
+import { OVERVIEW, ROUTES } from "@/utils/constants";
+import * as ICONS from "../../../assets/icons/side.menu";
+
+export const MENU_ITEMS = [
```

Comment:
Can we type it with the types defined above? That would guard us against type errors

Consider `interface MenuItem` from below